inji-web/src/__tests__/components/PageTemplate/Header.test.tsx

```diff
@@ -58,16 +58,16 @@ describe("Testing Header Container Functionality", () => {
     //     expect(mockedUsedNavigate).toHaveBeenCalled();
     // });
 
-    test('Check the presence of the Language Selector', () => {
-        renderWithProvider((<Header />));
-        const headerElementLi = screen.getByTestId("Header-Menu-LanguageSelector");
-        expect(headerElementLi).toBeInTheDocument();
-        const headerElementDiv = screen.getByTestId("Header-Menu-Help-div");
-        expect(headerElementDiv).toBeInTheDocument();
+    // test('Check the presence of the Language Selector', () => {
```

Comment:
why have we commented this test

Reply:
Removed it since it is not required for the latest changes